scripts/git-tag.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+# This script tag a PR with the earliest version that contains it
+# to avoid manual handling and time taking.
+
+# Sample usage:
+#		$ git-tag <PullRequestNumber> <Commit Sha>
+#
+# If the scripts shows: gh: Validation Failed (HTTP 422)
+# it is expected and the tag will be only attached and not created
+# even if we use silent, we cannot avoid this message to come from API.
+
+set -euo pipefail
+
+PRNUM=$1
+COMMITSHA=$2
+
+GIT_VERSION=$(git tag --contains ${COMMITSHA})
+
+# COMMENT="""
+# Automated comment to show tag version from the commits.
+# **********************
+# - Commit 08d4a089ffe33676a0fd421e024e244cbd1957fa
+# - tag version: ${GIT_VERSION}
+# **********************
+# """
+
+# gh pr comment 1 --body "${COMMENT}"
+
+LABEL="earliest-release-v${GIT_VERSION}"
+# First we need to create the label
+if gh api --silent repos/alan-mas/stringer_test/labels -f name=${LABEL} -f color="000000"
```

Comment:
Let's `curl` the GitHub API directly. The `gh` command isn't installed everywhere this may run.

Reply:
Done :)